contrib_bots/bots/weather/weather.py

```diff
@@ -0,0 +1,69 @@
+# See readme.md for instructions on running this code.
+import requests
+import json
+import os
+import sys
+from six.moves.configparser import SafeConfigParser
+
+
+DIRECTORY = os.path.dirname(os.path.realpath(__file__))
+if not os.path.exists(DIRECTORY + '/.weather_config'):
+    print('Weather bot config file not found, please set up it in /.weather_config'
```

Comment:
Instead of saying this

```
please set it up in /.weather_config
```

can you say something like this instead?

```
please set it up in .weather_config, in the same directory as this bot
```

(I'm pretty sure you don't want the user to put `.weather_config` in the root directory of the file system.)